eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/MultiSigExamples.hs

```diff
@@ -95,39 +97,48 @@ _assertScriptHashSizeMatchesAddrHashSize (ScriptHash h) =
   KeyHash (Hash.castHash h)
 
 -- Multi-signature scripts
-singleKeyOnly :: Era era => Addr (EraCrypto era) -> MultiSig era
+singleKeyOnly ::
+  (ShelleyEraScript era, NativeScript era ~ MultiSig era) =>
```

Comment:
How odd.. I'm sure I  added them under duress from the compiler rather than arbitrarily, but you're right, they're not needed... Not sure why that happened, maybe an intermediary state or something :( Bah . 
Thanks!